source/opencascade/utilities.cc

```diff
@@ -600,6 +600,13 @@ namespace OpenCASCADE
     Assert(props.IsCurvatureDefined(), ExcMessage("Curvature is not well defined!"));
     Standard_Real Mean_Curvature = props.MeanCurvature();
     Point<3> normal = Point<3>(Normal.X(),Normal.Y(),Normal.Z());
+
+    if (face.Orientation()==TopAbs_REVERSED)
+      {
+        normal *= -1;
+        Mean_Curvature *= -1;
+      }
```

Comment:
Do you mind to annotate this fix with a small comment and also provide a small entry in ```doc/news/changes.h```? :-)